pkg/daemon/daemon.go

```diff
@@ -54,6 +53,10 @@ const (
 	maxUpdateBackoff = 60 * time.Second
 )
 
+var (
+	defaultRebootTimeout = 24 * time.Hour
```

Comment:
24h seems long to me. can we make it shorter? 6h maybe?

Reply:
Took this from MCO. I don't see a shutdown event being held for that much time, so let's bring it down to 6h.